app/assets/javascripts/backlogs/burndown.js

```diff
@@ -63,8 +63,13 @@ RB.Burndown = (function ($) {
       }
       $('#charts').html("<div class='loading'>" + RB.i18n.generating_graph + "</div>");
 
-      var url = RB.urlFor('show_burndown_chart', { sprint_id: $(this).data('this').sprintId, project_id: RB.constants.project_id});
-      window.open(url);
+      var url = RB.urlFor('show_burndown_chart', {
+          sprint_id: $(this).data('this').sprintId,
+          project_id: RB.constants.project_id
+      });
+      
+      var burndownWindow = window.open(url);
+      burndownWindow.sessionStorage.clear();
```

Comment:
TODO: I don't get what this line is for as I am not deep enough into this plugin. Is this line still needed?

Reply:
Good one, it is not need any more 👍